src/Jackett.Common/Indexers/Torrentscsv.cs

```diff
@@ -97,11 +98,8 @@ public async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuery query, int
                     if (torrent == null)
                         throw new Exception("Error: No data returned!");
 
-                    var release = new ReleaseInfo();
-                    release.Title = torrent.Value<string>("name");
-
                     // construct magnet link from infohash with all public trackers known to man
-                    var magnet_uri = "magnet:?xt=urn:btih:" + torrent.Value<JToken>("infohash") +
+                    var magnet_uri = new Uri( "magnet:?xt=urn:btih:" + torrent.Value<JToken>("infohash") +
```

Comment:
Add a TODO, we should have a global var with public bittorent trackers so we can reuse it in all trackers.

Reply:
In all **public** trackers. Private trackers don't like when you share their stuff externally like that.